Resources/views/Resetting/checkEmail.html.twig

```diff
@@ -4,6 +4,6 @@
 
 {% block fos_user_content %}
 <p>
-{{ 'resetting.check_email'|trans({'%email%': email}) }}
+{{ 'resetting.check_email'|trans({'%username%': username}) }}
```

Comment:
this one breaks all the resetting.check_email messages which need to be refactored. I think the best approach would be to have a generic message like 

"An email has been sent to the email address associated with the account. It contains a link you must click to reset your password."